### DISCUSSION TITLE
i want set my skin to steve

### DISCUSSION THREAD

**Discussion by TrenchRat** [ORelio/Minecraft-Console-Client#2157] [Q&A] [answered]
how i change it to just steve not others skins(i in offline mode)

**Comment by ORelio** [comment]
I'm not sure you can do that.

**Reply by breadbyte** [reply]
I think they mean the application icon that shows up as the current head of the player's character?

**Comment by Splarkszter** [comment, marked as answer]
Line 44 in MinecraftClient.ini (AKA config file)

`playerheadicon=true # Only works on Windows XP-8 or Windows 10 with old console`
change `playerheadicon=true` to `playerheadicon=false`

**Reply by Splarkszter** [reply]
If you are talking about in-game skin. if you are on a server with some plugin that restores skins you can't controll that.
